engine/config/1.0/schema.json

```diff
@@ -127,9 +131,12 @@
                       "type": "object",
                       "properties": {
                         "limit": {
-                          "type": "integer",
-                          "default": 100,
-                          "description": "The maximum number of requests allowed per account per time unit."
+                          "description": "The maximum number of requests allowed per account per time unit.",
+                          "oneOf": [
+                            { "type": "integer", "default": 100 },
+                            { "$ref": "#/$defs/envVarPattern" },
+                            { "$ref": "#/$defs/filePattern" }
+                          ]
```

Comment:
Missed this in #1. Just like other ints in this file, values can be injected from env: or file: